src/lifetimes/exercise.rs

```diff
@@ -33,7 +34,7 @@ enum FieldValue<'a> {
     Varint(u64),
     //I64(i64),  -- not needed for this exercise
     Len(&'a [u8]),
-    I32(i32),
+    I32(f32),
```

Comment:
I don't think I understand why the PR changes this? From reading the discussion, I thought the PR was about giving students a chance to implement something more?